cmd/buildah/push.go

```diff
@@ -179,3 +215,32 @@ func getListOfTransports() string {
 	allTransports := strings.Join(transports.ListNames(), ",")
 	return strings.Replace(allTransports, ",tarball", "", 1)
 }
+
+// getAllTags gets all of the locally tagged images and adds them
+// to the passed in slice.
+func getAllTags(imageName string, imageTags *[]string, store storage.Store) error {
+
+	imageName = "/" + imageName
+	images, err := store.Images()
+	if err != nil {
+		return errors.Wrapf(err, "error reading images")
+	}
+	for _, image := range images {
+
+		if len(image.Names) < 1 {
+			continue
+		}
+		// Name should look like: "docker.io/library/alpine:latest"
+		for _, name := range image.Names {
+			splitName := strings.Split(name, ":")
+			if !strings.HasSuffix(splitName[0], imageName) {
+				break
+			}
+			if len(splitName) > 1 {
+				*imageTags = append(*imageTags, splitName[1])
+			}
+		}
+
+	}
+	return nil
```

Comment:
(I would find it a bit more natural for `imageTags` to be a return value instead of a reference parameter.)